application/testing/CMakeLists.txt

```diff
@@ -253,9 +258,7 @@ f3d_ss_test(NAME Version TEMPLATE ${_screenshot_dir}/{app}_{version}_{version_fu
 f3d_ss_test(NAME Model TEMPLATE ${_screenshot_dir}/{model}_{model.ext}_{model_ext}.png EXPECTED ${_screenshot_dir}/suzanne_suzanne.ply_ply.png)
 f3d_ss_test(NAME ModelN1 TEMPLATE ${_screenshot_dir}/{model}_{n}_{n:2}.png EXPECTED ${_screenshot_dir}/suzanne_1_01.png)
 f3d_ss_test(NAME ModelN2 TEMPLATE ${_screenshot_dir}/{model}_{n}_{n:2}.png EXPECTED ${_screenshot_dir}/suzanne_2_02.png DEPENDS TestScreenshotModelN1)
-string(TIMESTAMP DATE_Y "%Y")
-string(TIMESTAMP DATE_Ymd "%Y%m%d")
-f3d_ss_test(NAME Date TEMPLATE ${_screenshot_dir}/{model}_{date}_{date:%Y}.png EXPECTED ${_screenshot_dir}/suzanne_${DATE_Ymd}_${DATE_Y}.png)
+f3d_ss_template_test(NAME Date TEMPLATE ${_screenshot_dir}/{model}_{date}_{date:%Y}.png EXPECTED_REGEX ${_screenshot_dir}/suzanne_[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]_[0-9][0-9][0-9][0-9]\.png)
```

Comment:
since you use this only once I wonder if you really need a cmake function ? just declare the test using f3d_test ?

Reply:
I'd argue the function helps for readability. Maybe I can use it for some of the other tests to remove the "it's only used once" reservation (and insignificantly lighten the CI at the same time)?

Reply:
that would be awesome if you see a partern for improving the regex tests.

Reply:
> that would be awesome if you see a partern for improving the regex tests.

I just meant replacing some of the `f3d_ss_test` with `f3d_ss_template_test`. I'm not a `ctest` wizard yet :)